crates/daphne-service-utils/src/compute_offload/mod.rs

```diff
@@ -0,0 +1,499 @@
+// Copyright (c) 2025 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use crate::{
+    capnproto::{
+        decode_list, encode_list, usize_to_capnp_len, CapnprotoPayloadDecode,
+        CapnprotoPayloadEncode,
+    },
+    compute_offload_capnp::{
+        hpke_receiver_config::{self, hpke_config},
+        initialize_reports,
+        initialized_reports::{self, initialized_report},
+        partial_dap_task_config, prepare_init, report_metadata, time_range, vdaf_verify_key,
+    },
+};
+use daphne::{
+    constants::DapAggregatorRole,
+    hpke::{HpkeConfig, HpkeReceiverConfig},
+    messages::{self, HpkeCiphertext, PrepareInit, ReportMetadata, ReportShare, TaskId},
+    vdaf::{VdafConfig, VdafPrepShare, VdafPrepState, VdafVerifyKey},
+    InitializedReport, PartialDapTaskConfigForReportInit, WithPeerPrepShare,
+};
+use prio::codec::{Encode, ParameterizedDecode, ParameterizedEncode};
+use std::{borrow::Cow, ops::Range};
+
+pub struct InitializeReports<'s> {
+    pub hpke_keys: Cow<'s, [HpkeReceiverConfig]>,
+    pub valid_report_range: Range<messages::Time>,
```

Comment:
WDYM by valid here?

Reply:
Whatever we use to mean here: https://github.com/cloudflare/daphne/blob/974dd0c23092809ad7a8bd1b5715902bdd61a9d9/crates/daphne/src/roles/aggregator.rs#L134

Reply:
Cool, would you mind documenting this for future reference?
```suggestion
    /// Output of `DapAggregator::valid_report_time_range()` at the start of the aggregation job.
    pub valid_report_range: Range<messages::Time>,
```